Tests/Marketplace/marketplace_services.py

```diff
@@ -1476,7 +1476,8 @@ def collect_content_items(self):
                 PackFolders.GENERIC_DEFINITIONS.value: "GenericDefinitions",
                 PackFolders.GENERIC_FIELDS.value: "GenericFields",
                 PackFolders.GENERIC_MODULES.value: "GenericModules",
-                PackFolders.GENERIC_TYPES.value: "GenericTypes"
+                PackFolders.GENERIC_TYPES.value: "GenericTypes",
+                PackFolders.LISTS.value: "list"
```

Comment:
are we sure it's not capitalized?

Reply:
singulre and lowercase as can be seen here: 
https://github.com/demisto/server/blob/6a8074b45de1f8ccc53cd774dc02a9a1d6e95623/domain/contentpack.go#L224